misk/web/tabs/web-actions/src/containers/WebActionCardContainer.tsx

```diff
@@ -68,7 +68,7 @@ const WebActionCardContainer = (
               tag={`${props.tag}::Services`}
             />
             <MetadataCollapse
-              content={props.action.allowedRoles}
+              content={props.action.allowedCapabilities}
```

Comment:
@adrw can you double check all the web stuff. I basically just did a sed of roles -> capabilities and science dog'd it.

Reply:
Looks good from my initial testing and code review 👍